db/seeds.rb

```diff
@@ -4,7 +4,7 @@
 
 OPTIONS = {headers: true, header_converters: :symbol}
 
-CSV.foreach "data/weather.csv", OPTIONS do |row|
+CSV.foreach "db/fixture-data/weather.csv", OPTIONS do |row|
```

Comment:
@icorson3 @AliSchlereth, I am curious to how we could structure our seed file so that a seeding of fixture data is used only for production and the total dataset is used for local development? Any documentation or suggestions would be greatly appreciated.

Reply:
@mgmilton You can build custom rake task to build in production and use the seed file for the development environment.